src/bot/cogs/moderator.py

```
@@ -461,6 +482,18 @@ async def execute_automod(self, message: discord.Message, config: Dict, reason:
             except:
                 pass
 
+        if "kick" in actions:
+            try:
+                await message.author.kick(reason=reason)
+            except:
```

Ruff (0.15.7)

[error] Do not use bare `except`

(E722)